client/app/main/main.html

```diff
@@ -62,6 +62,14 @@
       review project proposals and decide which projects should receive
       financial support.
       </p>
+      <h3>Contact</h3>
+      <p>
+      For information and inquiry, please contact us via e-mail at <span id="contactmail">(javascript required)</span>.
+      </p>
+      <!-- If e-mail is not showing, please load main.js earlier -->
+      <script type="text/javascript">
+        getEmail();
```

Comment:
The entire page won't load if the user doesn't have javascript, so ```(javascript required) ``` is redundant